src/components/FiltersCollection.jsx

```diff
@@ -39,6 +88,24 @@ function FiltersCollection () {
         <Button className="me-1" variant="outline-primary" onClick={handleApplySearch}>Search</Button>
         <Button variant="outline-secondary" onClick={handleClearSearch}>Reset Search</Button>
       </ButtonToolbar>
+      <br></br>
+      <Form.Group >
+        <Form.Select value={itemType} onChange={handleCategory}>
+          <option>{DEFAULT_TYPE_OPTION}</option>
+          {itemTypeDropdowns}
+        </Form.Select>
+      </Form.Group>
+      <Form.Group >
+        <Form.Select value={itemRating} onChange={handleRating}>
+          <option>{DEFAULT_RATING_OPTION}</option>
+          {ratingDropdowns}
+        </Form.Select>
+      </Form.Group>
+      <br></br>
+      <a className = "searchBar">
```

Comment:
@cailors I'm getting a warning when running this, so I believe you just need to change this `a` tag to a `div`.  

The warning:
```
WARNING in src/components/FiltersCollection.jsx
  Line 105:7:  The href attribute is required for an anchor to be keyboard accessible. Provide a valid, navigable address as the href value. If you cannot provide an href, but still need the element to resemble a link, use a button and change it with appropriate styles

Reply:
fixed